content/04.results.md

```diff
@@ -47,21 +47,21 @@ The last most column consists of pooling every relationship specific label funct
 ![
 A grid of area under the receiver operating curve (AUROC) for each edge type.
 Each plot consists of adding a single label function on top of the baseline model.
-This label function emits a positive (top row) or negative (bottom row) label at specified frequencies, and performance at zero is equivalent to not having a randomly emitting label function.
+This label function emits a positive (shown in blue) or negative (shown in orange) label at specified frequencies, and performance at zero is equivalent to not having a randomly emitting label function.
 The error bars represent 95% confidence intervals for AUROC (y-axis) at each emission frequency.
-](https://raw.githubusercontent.com/danich1/snorkeling/f8962788e462b783be05a6dec5eec7fe0f0259e7/figures/gen_model_error_analysis/transfer_test_set_auroc.png){#fig:random_label_function_auroc}
+](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/gen_model_error_analysis/transfer_test_set_auroc.png){#fig:random_label_function_auroc}
 
 We observed that including one label function of a mismatched type to distant supervision often improved performance, so we evaluated the effects of adding a random label function in the same setting.
-We found that adding random noise did not usually improve performance (Figures {@fig:random_label_function_auprc} and {@fig:random_label_function_auprc}).
+We found that adding random noise did not usually improve performance (Figures {@fig:random_label_function_auroc} and {@fig:random_label_function_auprc}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Figure {@fig:random_label_function_auprc}).
 However, in general the performance changes were smaller than those observed with mismatched label types.
 
 ![
 A grid of area under the precision recall curve (AUPR) for each edge type.
 Each plot consists of adding a single label function on top of the baseline model.
-This label function emits a positive (top row) or negative (bottom row) label at specified frequencies.
+This label function emits a positive (shown in blue) or negative (shown in orange) label at specified frequencies.
 The error bars represent 95% confidence intervals for AUPR (y-axis) at emission frequency.
-](https://raw.githubusercontent.com/danich1/snorkeling/f8962788e462b783be05a6dec5eec7fe0f0259e7/figures/gen_model_error_analysis/transfer_test_set_auprc.png){#fig:random_label_function_auprc}
+](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/gen_model_error_analysis/transfer_test_set_auprc.png){#fig:random_label_function_auprc}
```

Comment:
Usually I suggest versioned links because when the image changes, oftentimes your manuscript should too.